E2E-Demos/Investment-Banking/Airflow/spark_etl_new.yaml

```diff
@@ -38,8 +38,8 @@ spec:
     memory: 1g
     serviceAccount: hpe-spark
     volumeMounts:
-      - name: hpedemo-user01-volume
-        mountPath: /mounts/hpedemo-user01-volume
+      - name: {{dag_run.conf.get("username", "hpedemo-user01")}}-volume
```

Comment:
hpedemo-user01 should be used for dev only I assume. Unless we're giving hpedemo-user01 as a fallback for some reason?